mep/people/tests/test_views.py

```diff
@@ -1047,11 +1047,18 @@ def test_member_map(self):
         assert 'mapbox_basemap' in response.context
         assert 'paris_overlay' in response.context
         # address of the library itself should be in context
-        assert response.context['library_address']['name'] == 'Shakespeare & Company'
+        assert response.context['library_address']['name'] == 'Shakespeare and Company'
```

Comment:
Are we _sure_ this is the correct name to use? :-) ;-)

Reply:
I sure hope so!!